webapp/sass/profile.scss

```diff
@@ -223,6 +224,13 @@ $pane-width: 6px;
     margin: 0 $pane-width;
     width: 50%;
 
+    .flamegraph-header {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      border: 2px solid lime;
```

Comment:
Lets remove the lime border here. I only have this in the screenshot to highlight the element of the page that I was referring to.